packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -77,17 +91,19 @@ function Tabs(props: Props): JSX.Element {
     }
   };
 
-  const handleKeydown = (event) => {
-    let focusElement;
+  const handleKeydown = (event: React.KeyboardEvent<HTMLLIElement>) => {
+    let focusElement: HTMLLIElement | null = null;
 
-    switch (event.keyCode) {
-      case keys.right: {
-        const nextTab = tabRefs.indexOf(event.target) + 1;
+    switch (event.key) {
+      case 'ArrowRight':
+      case 'Right': {
```

Comment:
Since we don't support IE, would it make sense to remove support for "Right" and "Left" key values?